data/transactions/logic/TEAL_opcodes_v8.md

```diff
@@ -14,27 +14,21 @@ Ops have a 'cost' of 1 unless otherwise specified.
 - Bytecode: 0x01
 - Stack: ..., A: []byte &rarr; ..., [32]byte
 - SHA256 hash of value A, yields [32]byte
-- **Cost**:
-    - 7 (v1)
-    - 35 (since v2)
+- **Cost**: 35
```

Comment:
I like this. No point in reminding people about old versions that had miscalculated costs!